build-deb

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
```

Comment:
Do all these files have to reside in the toplevel directory?
Many of these files seem very generic. Are they identical with files used for other software tools, or are they customized for p4c? If they are generic, is there a way to import them dynamically instead of hardwiring them in the repository?